config/environments/test.rb

```diff
@@ -13,7 +13,8 @@
   config.eager_load = false
 
   # Configure static file server for tests with Cache-Control for performance.
-  config.serve_static_files   = false
+  # Changed to true by rake rails:update
+  config.serve_static_files   = true
```

Comment:
c.f. https://stackoverflow.com/questions/17550747/rails-4-enabled-rails-to-serve-static-assets-is-it-correct-on-heroku